plugins/ern_v0.39.0+/react-native-screens_v2.10.1+/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "com.google.android.material:material:1.1.0",
+      "androidx.coordinatorlayout:coordinatorlayout:1.1.0"
```

Comment:
Do we also need `androidx.fragment:fragment:1.2.1` here?
https://github.com/software-mansion/react-native-screens/blob/master/android/build.gradle#L56

Reply:
Ahh, good question.
It's working for us without adding it, but that's probably because the dependency is added by another plugin 🤔 
Do you usually add all dependencies from the `build.gradle` file?

Reply:
Quickly inspected the generated container and the `lib/build.gradle` doesn't contain `androidx.fragment:fragment:1.2.1`. It could be implicitly imported by another dependency though...

Reply:
Yeah good question. Not 100% sure what adds the dependency. Normally the auto-generated plugin needs quite a bit of cleanup of the dependencies added (e.g. `react-native`, `appcompat` etc).

A quick check in the react-native-screens repo shows that the project will also compile with the explicit dependency removed! So I think we're good to merge this one. Can you please rebase the branch real quick?

Reply:
Done!

Reply:
Hey @LcTwisk - Sorry for the delay. I had another look, and it turns out `fragment` is a dependency of both `appcompat` and `coordinatorlayout`. That's why it works even without explicitly specifying the dependency.

However, the latest version of both these dependencies is `1.1.0`, referencing `fragment:1.1.0`, but `react-native-screens` references `fragment:1.2.1`. While there are no build errors, I think it's best if we add version 1.2.1 of fragment in the plugin config here as well, just to keep everything aligned. I don't know if `react-native-screens` relies on anything that has changed in fragment between 1.1.0 and 1.2.1.

When you amend the commit, can you also apply the other minor changes you can see in this commit please:
https://github.com/friederbluemle/electrode-native-manifest/commit/ab89b030f90ea1712af7e71d760d33f31b1c92d8

This aligns it with a plugin config generated with the latest version of ern. Thanks!